snapcraft/internal/repo/_platform.py

```diff
@@ -26,6 +26,7 @@
     '"elementary"',
     'debian',
     'neon',
+    'LinuxMint',
```

Comment:
This is wrong. It should be `linuxmint`. The value being checked is the `ID` field in `/etc/os-release`.